app/views/events/index.html.erb

```diff
@@ -25,6 +25,7 @@
         <td><%= link_to 'Show', event %></td>
         <td><%= link_to 'Edit', edit_event_path(event) %></td>
         <td><%= link_to 'Attend', attend_path(event) %></td>
+        <td><p><%= event.comments.count %> <%= (event.comments.count) == 1 ? 'Comment' : 'Comments'%></p></td>
```

Comment:
What about its 0 or `<1`?

Reply:
Pour 0, le comment prend un "s" mais c'est pour le 1 qu'il n'y a pas de "s".

Reply:
If there is no comment, it will say "0 Comments"?

Reply:
yes